app/i18n/it/index.php

```diff
@@ -60,6 +60,18 @@
 		'reader_view' => 'Modalità di lettura',
 		'rss_view' => 'Feed RSS',
 		'search_short' => 'Cerca',
+		'sort' => array(
+			'_' => 'Sorting criteria',	// TODO
```

Comment:
@Alkarex I'm sorry but I don't know how to translate it into Italian

Reply:
`Sorteercriteria` is `critère de tri(age)` in French. A quick search suggests LibreOffice translates that to `Criteri di ordinamento`.

Reply:
@Frenzie I also looked it up but the translation I am not convinced because the term `ordinamento` is a term that is used in Italian to refer to a set of laws

Reply:
I couldn't say. What does Windows (or whichever OS) use for the `Sort by` menu if you right click in a file manager, `Ordina per`?

![image](https://github.com/user-attachments/assets/c9a56284-2c71-4c27-9211-bd57361551cc)

Reply:
```suggestion
			'_' => 'Ordina per',
```
@Frenzie Thank you I hadn't thought of translating it that way

@Alkarex Thank you for correcting my error in the code I edited earlier